Add AIBC coin configuration

nickgsh opened the pull request:
This pull request adds the configuration file for the AIBC coin to the `coins` repository. 
- Added `aibc.json` configuration file for AIBC coin.

cipig commented:
coins need electrum servers in order to work in the GUIs

nickgsh commented:
Added two Electrum servers for GUI compatibility ,

cipig commented:
> Added two Electrum servers for GUI compatibility ,

you need to add them to `electrums/AIBC` file, like the other electrums from other coins
saw something else: the entry for the coin itself needs to be added to `coins` file, not as a separate file
`avg_blocktime` is in seconds... i doubt that AIBC has 0.5 seconds block time
are you sure the coin is working for swaps? what is the github repo of the wallet? want to at least check if it has BIP65... but would be better to actually test a swap with it

nickgsh commented:
i have add AIBC electrums/AIBC , added to coins file , corrected the avg_blocktime,
github repo of the wallet ,  https://github.com/nickgsh/AiBlockChain
Please let me know  if anything else need to be corrected.
thank you,

cipig commented:
there are no changes in the PR, it's still the same: https://github.com/KomodoPlatform/coins/pull/918/files
did you forgot to commit?

nickgsh commented:
sorry about that, can you check now please

cipig commented:
looks better, but you added AIBC 2 times in the coins file
the icon only needs to be added to `icons_original` directory, the one in `icons` will be generated by the github action script later, so please delete it
i tried to connect to the electrums, but 50002 is the port for WSS, not SSL
please add the port of SSL... the WSS port is only needed for the Web version and can be added as `ws_url` param to the entry... but Mobile and Desktop need the SSL or TCP port, those can't use WSS... you can look at eg the electrums/KMD file to see how it looks like

cipig commented:
found out that 50001 is the TCP port, so i could enable AIBC on one of my makers with that
question: could you send some AIBC to `ASpnM3ZJuEsHKmbQL6gcXbUgSK7uTTtkER`? i would like to do a test swap. the amount doesn't matter, just needs to be more then the dust setting :-)

nickgsh commented:
sent , thank you,

cipig commented:
> sent , thank you,

thanks, got it
please also address what i mentioned in previous comment: https://github.com/KomodoPlatform/coins/pull/918#issuecomment-1897368843

cipig commented:
swap was fine
```
[taker fee (KMD)](https://kmdexplorer.io/tx/dd5ce0310cc51e31a9342fae09f83543e2e5c6964113d1d90bba51f9c55965be)  
[maker payment (AIBC)](http://95.111.231.8:3001/tx/75ec5a4fd5f18649a0ba820aaa4c3790c1408e64ba4b6631a75d459579b4eb80)  
[taker payment (KMD)](https://kmdexplorer.io/tx/565870fbd99fd5fae6c16aa544685986d3b5c7db12cc213df5462e9fe5f7d62e)  
[taker payment spent (KMD)](https://kmdexplorer.io/tx/40f7c499b6896f81a184432e1f1ca0a9ca446daf92eb4e37d21a729704493d63)  
[maker payment spent (AIBC)](http://95.111.231.8:3001/tx/f67f1b4a8cf44de2531ab0bc1f6bbd8c69c2821d05cc754e6a6e7c9c1db5a12e)  
```
i edited coins file to remove the duplicate and also the electrum config

nickgsh commented:
Thank you, much appreciated

cipig reviewed (approved):
![image](https://github.com/KomodoPlatform/coins/assets/32116761/cfcf8e5b-6284-4939-81e5-eab67af4f599)

cipig commented:
@nickgsh are you blocking certain IPs on the electrum servers?
They are both reachable, but our github action script (https://github.com/KomodoPlatform/coins/blob/master/utils/scan_electrums.py) sees them as unreachable and delists AIBC because it thinks that all electrums are down.

nickgsh commented:
Had to reboot the servers. They are working now.On Jan 19, 2024, at 10:04 AM, cipig ***@***.***> wrote:﻿
@nickgsh are you blocking certain IPs on the electrum servers?
They are both reachable, but our github action script (https://github.com/KomodoPlatform/coins/blob/master/utils/scan_electrums.py) sees them as unreachable and delists AIBC because it thinks that all electrums are down.

—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you were mentioned.Message ID: ***@***.***>

nickgsh commented:
they are up now , they should not go down again

nickgsh commented:
really appreciate all your help,
 what other steps do I need to take to get AIBC listed in the Komodo wallets?

cipig commented:
> what other steps do I need to take to get AIBC listed in the Komodo wallets?

There are no further steps. AIBC will be in the upcoming release: https://github.com/KomodoPlatform/komodo-wallet-desktop/pull/2407
